api/host.go

```diff
@@ -24,16 +24,26 @@ type (
 		WindowSize   types.BlockHeight  `json:"windowsize"`
 
 		NumContracts     uint64         `json:"numcontracts"`
+		LostRevenue      types.Currency `json:"lostrevenue"`
 		Revenue          types.Currency `json:"revenue"`
 		StorageRemaining int64          `json:"storageremaining"`
 		UpcomingRevenue  types.Currency `json:"upcomingrevenue"`
+
+		RPCErrorCalls        uint64 `json:"rpcerrorcalls"`
+		RPCUnrecognizedCalls uint64 `json:"rpcunrecognizedcalls"`
+		RPCDownloadCalls     uint64 `json:"rpcdownloadcalls"`
+		RPCRenewCalls        uint64 `json:"rpcrenewcalls"`
+		RPCReviseCalls       uint64 `json:"rpcrevisecalls"`
+		RPCSettingsCalls     uint64 `json:"rpcsettingscalls"`
+		RPCUploadCalls       uint64 `json:"rpcuploadcalls"`
```

Comment:
why
I doubt it makes any difference performance-wise. Grouping objects makes them easier to understand and manipulate.